stable/cert-manager-setup/templates/v1upgrade.yaml

```
@@ -1,5 +1,5 @@
 {{- if .Values.installCertManager }}
-{{- if (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1alpha1") -}}
+{{- if (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1alpha1") }}
 --
```

I think the actual bug is here:
```suggestion
---
```